conf/messages.en

```diff
@@ -507,13 +504,13 @@ registrationConfirmation.print.link.text = Print this page
 individualIdentityNotConfirmed.title = We could not confirm your identity
 individualIdentityNotConfirmed.heading = We could not confirm your identity
 individualIdentityNotConfirmed.p1 = The information you have entered does not match our records. Check if you have entered the right details.
-individualIdentityNotConfirmed.h2.1 = Try entering your NINO again
-individualIdentityNotConfirmed.p2 = If entering your NINO again does not work, you can
-individualIdentityNotConfirmed.p2.link.text = find your NINO
+individualIdentityNotConfirmed.h2.1 = Try entering your National Insurance number again
+individualIdentityNotConfirmed.p2 = If entering your National Insurance number again does not work, you can
+individualIdentityNotConfirmed.p2.link.text = find your National Insurance number
 individualIdentityNotConfirmed.h2.2 = Try entering your name again
 individualIdentityNotConfirmed.p3 = Check if you have entered your first name and last name correctly.
 individualIdentityNotConfirmed.h2.3 = Contact HMRC to help us find you
-individualIdentityNotConfirmed.p4 = If you have checked and entered your NINO and name again and we still cannot find your business, you can
+individualIdentityNotConfirmed.p4 = If you have checked and entered your National Insurance number and name again and we still cannot find your business, you can
```

Comment:
Should be updated to:
individualIdentityNotConfirmed.p4 = If you have checked and entered your National Insurance number and name again and we still cannot find you, you can